mlir/include/mlir/ExecutionEngine/SparseTensor/Storage.h

```diff
@@ -334,8 +301,7 @@ class SparseTensorStorage final : public SparseTensorStorageBase {
   /// Partially specialize lexicographical insertions based on template types.
   void lexInsert(const uint64_t *lvlCoords, V val) final {
     assert(lvlCoords);
-    // TODO: get rid of this! canonicalize all-dense "sparse" array into dense
-    // tensors.
+    // TODO: needed?
```

Comment:
Depending on whether we still want to handle "all-dense sparse" tensor. If so, I think we can remove the TODO here.